cmd/restore/restore.go

```diff
@@ -52,12 +52,17 @@ func NewRestoreCmd() *cobra.Command {
 			}
 			return c.restore(args[0])
 		},
-		PreRunE: preRunValidateConfig,
 	}
 
 	cmd.SilenceUsage = true
-	cmd.Flags().StringVar(&restoredConfigPath, "config-out", "", "Specify desired name and full path for the restored k0s.yaml file (default: ${cwd}/k0s_<archive timestamp>.yaml)")
-	cmd.Flags().AddFlagSet(config.FileInputFlag())
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil
```

Comment:
Returning `nil` in case `os.Getwd()` return error seems bit off

Reply:
I don't see why it needs it in the first place. It could just `return fmt.Sprintf("k0s_%s.yaml", fName)`, the working directory is assumed when paths are relative.